packages/streams/src/decorators.ts

```diff
@@ -0,0 +1,18 @@
+import type { Model, Program, Type } from "@typespec/compiler";
+import type { StreamOfDecorator } from "../generated-defs/TypeSpec.Streams.js";
+import { StreamStateKeys } from "./lib.js";
+
+/** @internal */
+export const namespace = "TypeSpec.Streams";
+
+export const $streamOf: StreamOfDecorator = (context, target, type) => {
```

Comment:
so I think now that we have teh `$decorators` the pattern shoiulkd be to name your decorator 

```
streamOfDecorator
```

just makes it clear now that it isn't getting picked up directly by typespec.